### PR TITLE
bump ("elm-version": "0.19.1") :: Rename examples

### PR DESCRIPTION
**Pull request by RalfBarkow**

### CODE REVIEW

File: examples/Crate.elm [hunk 1]
```diff
@@ -0,0 +1,305 @@
+module Crate exposing (main)
```
Comment (w0rm) [inline examples/Crate.elm]: This one doesn't come up as being renamed. Have you forgotten to delete the old `crate.elm`?

Comment (RalfBarkow) [reply examples/Crate.elm]: Yes, Andrey the Crate one doesn't come up as being renamed, but I have no idea why. Sorry.

Comment (w0rm) [reply examples/Crate.elm]: In your branch https://github.com/RalfBarkow/webgl/tree/develop/examples you still have the old `crate.elm` and the new `Crate.elm`.

**Comment by w0rm** [comment]
@RalfBarkow thanks!

**Comment by RalfBarkow** [comment]
@w0rm You are welcome!

**Pull request merged**